addons/mail/static/src/components/link_preview_card/link_preview_card.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-name="mail.LinkPreviewCard" owl="1">
+        <div class="o_LinkPreviewCard bg-300 rounded px-3 pt-3 pb-2 d-flex position-relative" t-attf-class="{{ className }}" t-ref="root">
+            <t t-if="linkPreviewCardView.linkPreview">
+                <div t-if="linkPreviewCardView.linkPreview.title" class="o_LinkPreviewCard_content">
+                    <a t-att-href="linkPreviewCardView.linkPreview.url" target="_blank">
+                        <h5 class="o_LinkPreviewCard_title" t-esc="linkPreviewCardView.linkPreview.title"></h5>
+                    </a>
+                    <p t-if="linkPreviewCardView.linkPreview.description" class="o_LinkPreviewCard_description" t-esc="linkPreviewCardView.linkPreview.description"></p>
+                    <img t-if="linkPreviewCardView.linkPreview.image_url" class="o_LinkPreviewCard_image img-fluid rounded mr-3 mb-2" t-att-src="linkPreviewCardView.linkPreview.image_url" t-att-alt="linkPreviewCardView.linkPreview.title"/>
```

Comment:
Why are all the other values depending on title? Can't each have their condition on themselves?